ftrip.io.catalog-service/ftrip.io.catalog-service/Accommodations/AccommodationsController.cs

```diff
@@ -13,23 +17,76 @@ namespace ftrip.io.catalog_service.Accommodations
     public class AccommodationsController : ControllerBase
     {
         private readonly IMediator _mediator;
+        private readonly CurrentUserContext _currentUserContext;
 
-        public AccommodationsController(IMediator mediator)
+        public AccommodationsController(IMediator mediator, CurrentUserContext currentUserContext)
         {
             _mediator = mediator;
+            _currentUserContext = currentUserContext;
         }
 
         [HttpGet("{id}")]
-        public async Task<IActionResult> ReadById(Guid id, CancellationToken ct = default)
+        public async Task<IActionResult> ReadById(Guid id, [FromQuery] bool s, CancellationToken ct = default)
         {
-            return Ok(await _mediator.Send(new ReadByIdQuery() { Id = id }, ct));
+            return Ok(await _mediator.Send(new ReadByIdQuery { Id = id, Simple = s }, ct));
         }
 
         [HttpPost]
         public async Task<IActionResult> Create(CreateAccommodationRequest request, CancellationToken ct = default)
         {
+            request.HostId = Guid.Parse(_currentUserContext.Id);
             return Ok(await _mediator.Send(request, ct));
         }
+
+        [HttpPut("{id}")]
+        public async Task<IActionResult> Update([FromRoute] Guid id, UpdateAccommodationRequest request, CancellationToken ct = default)
```

Comment:
You don't really need this attribute but if you want to be explicit that is also fine.